app/models/profile.rb

```diff
@@ -4,4 +4,14 @@ class Profile < ActiveRecord::Base
   validates :username, presence: true, on: :update
   validates :username, uniqueness: true, on: :update
   validates :user, presence: true
+
+  mount_uploader :avatar_url, ImageUploader
+
+  validates_processing_of :avatar_url
+  validate :avatar_url_size_validation
+
+  private
```

Comment:
but it does... there's two blank lines before and after private, line 12 and line 14?